cdc/entry/mounter.go

```diff
@@ -50,8 +50,10 @@ type baseKVEntry struct {
 
 type rowKVEntry struct {
 	baseKVEntry
-	Row    map[int64]types.Datum
-	PreRow map[int64]types.Datum
+	Row         map[int64]types.Datum
+	PreRow      map[int64]types.Datum
+	RowExist    bool
+	PreRowExist bool
```

Comment:
recommend adding some comments about why we add these two variables

Reply:
done